handlers/main.yml

```diff
@@ -6,19 +6,15 @@
   service: name=fstrim.timer state=enable
 
 - name: Reboot server
-  shell: sleep 2 && init 6
+  shell: shutdown -r now
```

Comment:
In version 2.7 there is now a command to reboot the Linux servers.
https://www.jeffgeerling.com/blog/2018/reboot-and-wait-reboot-complete-ansible-playbook

Reply:
This would be much better and also needs less handlers.